### DISCUSSION TITLE
Incorrect rendering of shoreline

### DISCUSSION THREAD

**Discussion by orientrun** [osmandapp/OsmAnd#15124] [Maps Related]
Strange behavior while using the latest Europe/Turkey/Mediterranean Region map: correctly displayed shoreline changes into straight one after slight shift in position (without zoom in/out).
No such problem with other installed maps.
OsmAnd+ 4.2.7 on Pixel 3a.
![CorrectShoreline](https://user-images.githubusercontent.com/111636669/185745376-2be62608-658e-4267-932d-d5562ebc6e1a.png)
![StraightShoreline](https://user-images.githubusercontent.com/111636669/185745403-5a2e7db5-0f8b-4358-b61d-f354732ae29d.png)

**Comment by vshcherb** [comment]
Going to check it. We have important fix

**Reply by vshcherb** [reply]
https://github.com/osmandapp/OsmAnd-Issues/issues/1074

**Reply by orientrun** [reply]
Page not found error
